### PR TITLE
Added profile info to transaction history

### PR DESCRIPTION
**Pull request by musidlo**
image

### CODE REVIEW

File: packages/app/features/profile/screen.tsx [hunk 1]
```diff
@@ -51,156 +55,193 @@ export function ProfileScreen({ sendid: propSendid }: ProfileScreenProps) {
...
-      ) : null}
-    </YStack>
+      {isProfileInfoVisible && (
+        <YStack
+          w={'100%'}
+          ai={'center'}
+          $gtLg={{
+            width: '35%',
+            minWidth: 400,
+            height: '81vh',
```
Comment (youngkidwarrior) [inline packages/app/features/profile/screen.tsx:226]: Make sure we only use vh when we sure we are on the web.

Either use the `$platform-web` prop or import the `isWeb` flag

File: packages/app/features/profile/screen.tsx [hunk 2]
```diff
@@ -267,3 +308,100 @@ const DatePill = ({ date }: { date: string }) => {
...
+                top={0}
+                left={0}
+                width="100%"
+                height="100%"
+                p={'$5'}
+                justifyContent="flex-end"
+                gap={'$3'}
+              >
+                <Paragraph size={'$9'} $theme-light={{ color: '$white' }}>
+                  Big investor
```
Comment (youngkidwarrior) [inline packages/app/features/profile/screen.tsx:367]: ? Haven’t seen the designs but this doesn’t look right

**Review by youngkidwarrior — CHANGES_REQUESTED** [review]
Just make sure we check for web when using vh

Make sure `Big Investor` is correct

File: packages/app/features/profile/screen.tsx (hunk 2 above)
Comment (musidlo) [reply packages/app/features/profile/screen.tsx:367]: lol

File: packages/app/features/profile/screen.tsx (hunk 1 above)
Comment (musidlo) [reply packages/app/features/profile/screen.tsx:226]: Im using vh only in $gtLg breakpoint, it wont apply for mobile

File: packages/app/features/profile/screen.tsx (hunk 2 above)
Comment (musidlo) [reply packages/app/features/profile/screen.tsx:367]: applied

File: packages/app/features/profile/screen.tsx (hunk 1 above)
Comment (musidlo) [reply packages/app/features/profile/screen.tsx]: discussed online, applied

**Review by youngkidwarrior — APPROVED** [review]

**Pull request merged**